docs/dagw_reddit.md

```diff
@@ -0,0 +1,97 @@
+
+# DAGW$_{DFM}$
+
+*Version*: 1.0.0
+
+*Homepage*: https://github.com/centre-for-humanities-computing/danish-foundation-models
+
+*license*: See the respective dataset
+
+
+DAGW$_{DFM}$ is a variant of Danish Gigaword [3] which excludes the sections containing
+tweets and modified news contained in danavis20. Twitter were excluded as an it was a
+sample of an dataset whiich was available to the authors. DanAvis20 (or danavis) were
+excluded due to preprocessing desribed in [3] (version 1 on 
+[arxiv](https://arxiv.org/pdf/2005.03521v1.pdf)) including shuffling of sentences,
+pseudonymization of prober names and the replacement of infrequent content wordswith
+statistical cognates, which could lead sentences such as *"Der er skilsmissesager i
+forsikringsselskabet"*.
```

Comment:
Assume it is this is causing the problem:

> replacement of infrequent content words with
statistical cognates

words that are infrequent within the given content (not sure exactly how this is done) and statistical cognates are words that have similar statistical properties. The word choice here matches the description in the DAGW paper (which is intended to avoid interpretation)